gpiozero/boards.py

```diff
@@ -59,15 +59,15 @@ def on(self):
         Turn all the output devices on.
         """
         for device in self:
-            if isinstance(device, (OutputDevice, CompositeOutputDevice)):
+            if isinstance(device, (OutputDevice, CompositeOutputDevice, RGBLED)):
```

Comment:
I don't like this. @waveform80 @lurch do you think it's possible for `RGBLED` to extend `OutputDevice` or `CompositeOutputDevice` rather than just `Device`?

Reply:
Actually I got around this by implementing on/off/toggle directly in RGBLEDBoard rather than using the parent class. Just working on getting blink to work too.

Reply:
IIRC @waveform80 implemented `RGBLED` as a child of `Device` rather than as a child of `OutputDevice` or `CompositeOutputDevice` because the `value` of an RGBLED is a 3-tuple and not a boolean or 0 -> 1 float.
And yup, RGBLED also implements it's own on / off / toggle / blink / pulse methods.